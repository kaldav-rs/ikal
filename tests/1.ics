BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
END:VCALENDAR
