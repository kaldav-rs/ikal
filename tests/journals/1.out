Ok(
    VJournal {
        dtstamp: Local(
            1997-09-01T15:00:00+02:00,
        ),
        uid: Text {
            params: {},
            text: "19970901T130000Z-123405@example.com",
        },
        class: None,
        created: None,
        dtstart: Date(
            1997-03-17,
        ),
        last_modified: None,
        organizer: None,
        recurid: None,
        sequence: None,
        status: None,
        summary: Some(
            Text {
                params: {},
                text: "Staff meeting minutes",
            },
        ),
        url: None,
        rrule: None,
        attach: [],
        attendee: [],
        categories: [],
        comment: [],
        contact: [],
        description: [
            Text {
                params: {},
                text: "1. Staff meeting: Participants include Joe\\,Lisa\\, and Bob. Aurora project plans were reviewed.There is currently no budget reserves for this project.Lisa will escalate to management. Next meeting on Tuesday.\\n2. Telephone Conference: ABC Corp. sales representativecalled to discuss new printer. Promised to get us a demo byFriday.\\n3. Henry Miller (Handsoff Insurance): Car wastotaled by tree. Is looking into a loaner car. 555-2323(tel).",
            },
        ],
        exdate: [],
        related_to: [],
        rdate: [],
        rstatus: [],
        x_prop: {},
        iana_prop: {},
    },
)
